twins/node.go

```diff
@@ -164,3 +160,10 @@ func newNode(n *Network, nodeID NodeID, consensusName string) (*node, error) {
 	}
 	return node, nil
 }
+
+func (n *node) EffectiveView() hotstuff.View {
```

Comment:
Not sure I quite understand the meaning of effective here. Could we add a doc comment?

The node has an `effectiveView` that is different from the node's `viewStates.View()`?